docs/_sass/_media.scss

```diff
@@ -23,6 +26,13 @@
 	.footer-resources {
 		width: 35%
 	}
+	.sub-card {
+		display: flex;
+    	flex-direction: column;
+	}
+	.col-md-4 {
```

Comment:
I'd change this to be .about-subitem instead of .col-md-4 so that this doesn't change the behavior of any other Bootstrap grid elements on the site. (pushed change to branch)